source/install/troubleshooting.rst

```diff
@@ -629,3 +631,300 @@ To help us narrow down whether it’s a server configuration issue, device speci
 
 - If you have another mobile device available, try connecting with that to see if your issue still reproduces.
 - If you don’t have another device available, check with other teammates to see if they are having the same issue.
+
+Opening a Support Ticket for Self-Managed Deployments
+-----------------------------------------------------
+
+When opening a Support ticket, it's important that you provide us with as much information as you can in a timely manner. Knowing what information is relevant can be confusing. We use the anagram C.L.U.E.S. to remember what we need:
+
+* Configurations
+* Logs
+* Users affected
+* Environment
+* Steps to reproduce
+
+C.L.U.E.S. represents all of the information that can clarify your issue. With these details, we can begin searching for a cause, whether it's a simple configuration change or a product bug. It also helps us when we need to escalate the issue to our developers so they can spend as much time as possible improving our product.
+
+Enterprise Edition customers have Premier Support and can open a support ticket in the `Enterprise Edition Support portal <https://mattermost.zendesk.com/hc/en-us/requests/new>`_. Additionally, peer-to-peer support is available for all Mattermost users on our `Community server <https://community-release.mattermost.com/core/channels/peer-to-peer-help>`_ and the `Troubleshooting forum <https://forum.mattermost.org/c/trouble-shoot>`__.
```

Comment:
We should use the default community.mattermost.com server here as it's the stable server.